### ISSUE TITLE
The new master is orphaned

### ISSUE THREAD

**Issue by mindojo-victor** [flupke/rainbow-saddle#14]
I have this supervisor config:
```ini
[program:brain]
command = rainbow-saddle --pid %(here)s/run/gunicorn.pid gunicorn --config=python:settings.gunicorn brain.application:application
```
When I start supervisor:
![image](https://user-images.githubusercontent.com/17159503/34479447-a2c0fa8c-efaf-11e7-814a-9baaa7c70e44.png)

When I send HUP signal to rainbow-saddle, it sends USR2 signal to Gunicorn, which forks into a new master. Then old master is killed, making the new master orphaned.

**Is this the correct/expected behavior?**

**Comment by flupke** [comment]
Hi, and sorry for the late reply.

I don't use gunicorn or rainbow-saddle in production anymore, I don't even do web development, so it's a bit hard to follow on issues, sorry.

I wrote rainbow-saddle for an earlier version of gunicorn which had different behavior, and compatibility with newer versions has been contributed by other developers.

I'm not sure what to do with this project, people seem to still be using it, but it bothers me to maintain it without using it.

**Comment by mindojo-victor** [comment]
I guess you should change README mentioning that the project in closed/unsupported. Then someone could take over it.
